### DISCUSSION TITLE
Our Impact!

### DISCUSSION THREAD

**Discussion by NoahGWood** [NoahGWood/OpenEFT#23] [Announcements]
Hi all!

I'm excited to announce the impact OpenEFT is having! Based on the number of clones, we've helped over 8,500 people generate their EFT Files for **FREE**! In the two months since starting this project, we have helped Americans save over $570,000 on their NFA forms!

Check out our infographic!
https://infogram.com/openeftorg-impact-infographic-1hxr4zxqke10q6y?live

**Comment by Tim-Healey** [comment]
@NoahGWood this is awesome. Thanks for all the effort you put into this project. Thanks to everyone else helping answer questions from people new to github/linux. Sucks that we have to do all this work (Thanks ATF) but this project makes the process easier/more affordable for literally thousands.
